rainbowsaddle/__init__.py

```diff
@@ -12,6 +12,14 @@
 import functools
 import traceback
 
+stats = None
+
+try:
+    import statsd
+    stats = statsd.StatsClient(prefix='rainbowsaddle')
+except ImportError:
+    print('Statsd import failed, statsd will be diabled', file=sys.stderr)
```

Comment:
If statsd is not installed, this will always print an error, correct? Can we gracefully handle this case for upstreaming?